ocs_ci/utility/utils.py

```diff
@@ -2574,3 +2574,27 @@ def check_for_rhcos_images(url):
     """
     r = requests.head(url)
     return r.status_code == requests.codes.ok
+
+
+def skipif_upgraded_from(version_list):
+    """
+    This function evaluates the condition to skip a test if the cluster
+    the upgraded from a particular OCS version
+
+    Args:
+        version_list (list): List of versions to check
+
+    Return:
+        (bool): True if test needs to be skipped else False
+    """
+    from ocs_ci.ocs.resources.ocs import get_ocs_csv
+    skip_this = False
+    version_list = [version_list] if isinstance(version_list, str) else version_list
+    ocs_csv = get_ocs_csv()
+    csv_info = ocs_csv.get()
+    prev_version = csv_info.get('spec').get('replaces', '')
+    for version in version_list:
+        if f'.v{version}' in prev_version:
+            skip_this = True
+            break
+    return skip_this
```

Comment:
To be on safe side and as it's called in collection of tests and I think that there can be the case that the OCS is not installed we should wrap all this to general try: except: Exception block.  In error message write the log message if something happen and return false.

Reply:
Done